src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.ts

```diff
@@ -772,17 +774,27 @@ export class QuizParticipationComponent implements OnInit, OnDestroy {
      * Callback method to be triggered when the user changes any of the answers in the quiz (in sub components based on the question type)
      */
     onSelectionChanged() {
-        this.applySelection();
-        if (this.sendWebsocket) {
-            if (!this.disconnected) {
-                // this.isSaving = true;
-                this.submission.submissionDate = this.serverDateService.now();
-                this.sendWebsocket(this.submission);
-                this.unsavedChanges = false;
-                this.updateSubmissionTime();
-            } else {
-                this.unsavedChanges = true;
-            }
+        this.unsavedChanges = true;
+    }
+
+    triggerSave(resetAutoSaveTimer = true): void {
+        if (resetAutoSaveTimer) {
+            this.autoSaveTimer = 0;
+        }
+        if (this.unsavedChanges && !this.isSubmitting) {
+            this.applySelection();
+            this.submission.submissionDate = this.serverDateService.now();
+            this.quizParticipationService
+                .saveOrSubmitForLiveMode(this.submission, this.quizId, false)
+                .pipe(take(1))
+                .subscribe({
+                    next: () => {
+                        // TODO: Do we need the response for anything here?
```

Comment:
Open TODO? I think that this should be resolved before a merge

Reply:
As with save via websocket, we did not end up needing the response here. I removed the comment